flytekit/clis/sdk_in_container/register.py

```diff
@@ -36,20 +36,18 @@ def register_tasks_only(project, domain, pkgs, test, version):
         t.register(project, domain, _utils.fqdn(m.__name__, k, entity_type=t.resource_type), version)
 
 
+
```

Comment:
Also, since you have a PR open, can you fix another bug we have here with the test flag.  You'll notice in register_all and register_tasks_only, we only print that we aren't actually doing anything with registration.  But that's a lie :p, we go ahead and register things.

So could you change the register calls to be like?
```
if not test:
    o.register(...)
```

Reply:
See #37 :)

Reply:
!!

Reply:
Anyway, this will clear my 'approval' after you push another commit--no matter how small--so please just remove that extra blank line and I will approve straight away!